client/proto/concord_client_request/v1/concord_client_request.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 VMware, all rights reserved
```

Comment:
All protobuf files in here are the external interface of the "client" which effectively is the clientservice. `ConcordClientRequest` doesn't fit here because it is an internal message (communication between concord client and concord).
I suggest to move the message definition somewhere under `client/concordclient`.

Reply:
Moved to client/concordclient directory